docs/modules/mssql.md

```diff
@@ -2,15 +2,17 @@
 
 _Note: this module is PREVIEW. It is under moderate development, but future releases may still break the API._
 
-Open Service Broker for Azure (OSBA) contains three Azure SQL Database services. These services enable you to select the most appropriate provisioning scenario for your needs. These services are:
+Open Service Broker for Azure (OSBA) contains five Azure SQL Database services. These services enable you to select the most appropriate provisioning scenario for your needs. These services are:
 
 | Service Name | Description |
 |--------------|-------------|
 | `azure-sql` | Provision both a SQL Server DBMS and a database. |
 | `azure-sql-dbms` | Provision only a SQL Server Database Management System (DBMS). This can be used to provision multiple databases at a later time. |
 | `azure-sql-database` | Provision a new database only upon a previously provisioned DBMS. |
+| `azure-sql-dbms-registered` | Register an existing server as a DBMS service instance. |
```

Comment:
I suggest put migration and dr related services to another doc. Otherwise the customer will be a little confused about these services.

Reply:
Good point. What about a new md named *mssql-advanced*?

Reply:
By the way, I didn't give `azure-sql-dbms-registered` a migration tag because I think it is a common scenario. Only `azure-sql-database-from-existing` in this module has the migration tag. Does it make sense?

Reply:
I rethought this. We said users could be confused when `cf marketplace` lists too many services. Here is the document for a certain service module. It is everything about the module. They look for what they want themselves. Do users really care about those services if they don't even see them in the `cf marketplace`?

Reply:
If we don't exclude it from the general doc, customers will get 4 mssql related services when they run `cf marketplace`, but when they refer to the doc, they will find 5 mssql related services. If I were the customer, I would be really confused, doubting whether I have mis-configured something...

Reply:
I'm not familiar with MASB usage, but it sounds reasonable to make `azure-sql-dbms-registered` as a general service.

Reply:
For `azure-sql-database-from-existing`, the description includes "The service requires `ENABLE_MIGRATION_SERVICES` to be `true` in OSBA environment variables". I think it is clear enough why the user didn't see this service in cf marketplace.

Reply:
For `azure-sql-database-from-existing`, the description includes "The service requires `ENABLE_MIGRATION_SERVICES` to be `true` in OSBA environment variables". I think it is clear enough why the user didn't see this service in cf marketplace.